pkg/spec/types.go

```diff
@@ -144,6 +145,10 @@ type SecretMod struct {
 
 // ControllerFields are the common fields in every controller Kedge supports
 type ControllerFields struct {
+	// Field to specify the version of application
+	// +optional
+	Appversion json.Number `json:"appversion,omitempty"`
```

Comment:
It would be better to have `Appversion` as a string. We should allow appversions like `1.0.4-beta1`